app/models/patron_request.rb

```diff
@@ -46,16 +46,52 @@ def default_service_point_code
     service_points.first || Settings.folio.default_service_point
   end
 
-  def origin_library_code
-    folio_location&.library&.code
+  def mediateable?
+    items_in_location.all?(&:mediateable?)
```

Comment:
This is a tiny thing, but I think we should error on the side of making things mediated if there's any doubt (and no need to check 1000 items if 1 will do?):

```suggestion
    items_in_location.any?(&:mediateable?)
```